<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.dell.delta_uat_automation.testcases.TC_1341382_LA_Dynamic_Dispatch_CUSTOM_SKU_PartsAndLabor_UX" tests="1" time="220.754" ignored="0" failures="1" timestamp="14 Mar 2017 08:58:45 GMT" skipped="0" hostname="W1041LQD02" errors="0">
  <testcase name="validateTC_1341803_EMEA_Happy_Path_PartsAndLabor_UX" time="220.754" classname="com.dell.delta_uat_automation.testcases.TC_1341382_LA_Dynamic_Dispatch_CUSTOM_SKU_PartsAndLabor_UX">
    <failure type="java.lang.AssertionError" message="Caught Error in Creation SR :">
      <![CDATA[java.lang.AssertionError: Caught Error in Creation SR :
	at org.testng.Assert.fail(Assert.java:83)
	at com.dell.delta_uat_automation.workflows.AssetsWorkFlowUX.SR_CreationUX(AssetsWorkFlowUX.java:1060)
	at com.dell.delta_uat_automation.testcases.TC_1341382_LA_Dynamic_Dispatch_CUSTOM_SKU_PartsAndLabor_UX.validateTC_1341803_EMEA_Happy_Path_PartsAndLabor_UX(TC_1341382_LA_Dynamic_Dispatch_CUSTOM_SKU_PartsAndLabor_UX.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.InvalidElementStateException: Element must not be hidden, disabled or read-only (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 183 milliseconds
Build info: version: 'unknown', revision: '31c43c8', time: '2016-08-02 21:57:56 -0700'
System info: host: 'W1041LQD02', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_102'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=false, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=11, platform=WINDOWS, nativeEvents=false, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:31382/, javascriptEnabled=true, ignoreProtectedModeSettings=true, enableElementCacheCleanup=true, unexpectedAlertBehaviour=accept}]
Session ID: 1b2573fc-7d78-4b1d-bd8e-f847bf62054f
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:683)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:319)
	at org.openqa.selenium.remote.RemoteWebElement.clear(RemoteWebElement.java:140)
	at com.dell.delta_uat_automation.workflows.AssetsWorkFlowUX.SR_CreationUX(AssetsWorkFlowUX.java:1021)
	... 26 more
]]>
    </failure>
  </testcase> <!-- validateTC_1341803_EMEA_Happy_Path_PartsAndLabor_UX -->
</testsuite> <!-- com.dell.delta_uat_automation.testcases.TC_1341382_LA_Dynamic_Dispatch_CUSTOM_SKU_PartsAndLabor_UX -->
